racingcar-java/src/main/java/RacingCar.java

```diff
@@ -0,0 +1,36 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RacingCar {
+
+    List<Car> cars;
+
+    public RacingCar(int numberCars) {
+        this.cars = setCars(numberCars);
+    }
+
+    List<Car> setCars(int numberCar) {
```

Comment:
```suggestion
    private List<Car> setCars(int numberCar) {
```
접근제어자는 default보다 private을 사용하는 것이 현재 막으려는 접근 범위에 더 맞겠네요